src/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -85,3 +85,9 @@ pscanrules.testinfosessionidurl.refs = http://seclists.org/lists/webappsec/2002/
 pscanrules.testinfosessionidurl.referrer.alert = Referer Exposes Session ID
 pscanrules.testinfosessionidurl.referrer.desc = A hyperlink pointing to anther host name was found. As session ID URL rewrite is used, it may be disclosed in referer header to external hosts.
 pscanrules.testinfosessionidurl.referrer.soln = This is a risk if the session ID is sensitive and the hyperlink refers to an external or third party host. For secure content, put session ID in secured session cookie.
+
+pscanrules.fingerprintingheaderscanner.name = Fingerprinting Headers
```

Comment:
Resource keys should be alpha sorted (we know there aren't all as things currently stand but that is the general plan/want).

I haven't reviewed the content of these as I'm anticipating it changing with the next PR.